### DISCUSSION TITLE
Correct folder contents and proper order on SD Card for 2.10.1

### DISCUSSION THREAD

**Discussion by JrBoy66** [EdgeTX/edgetx#5162] [Q&A] [answered]
WARNING! Newbie question: I've finally got my SD Card contents up to date after improperly upgrading my TX16s (org. ver.) from OpenTX. 
Is there a reference, e.g., a graphic or a list of the correct folder contents and order of the files?  I believe some of my escapades have resulted in duplicate folders and mis filed contents.  Specifically, my Wizard widget only shows glider, plane, and quad.  No Heli.  The files are present with the proper scripts but doesn't give the option in the form of a widget.

**Comment by pfeerick** [comment, marked as answer]
For the TX16S, have a look at https://github.com/EdgeTX/edgetx-sdcard/releases/download/latest/c480x272.zip for the raw SD card content  (this is just the SD card content, the voice/sound pack is extra). For the wizard, you probably want to focus on the TEMPLATES folder...

**Reply by JrBoy66** [reply]
Thank You. Guess I am being a little to OCD. Makes since! Thanks again for all your hard work and help! ✌️
